docs/examples.rst

```diff
@@ -853,7 +853,46 @@ exactly is happening during test execution. This also allows to
 inspect particular place of the code by inserting a ``bash`` in
 the shell code or ``import pdb; pdb.set_trace()`` for python::
 
-    tmt run --all execute --how beakerlib.tmt --interactive
+    tmt run --all execute --how tmt --interactive
+
+
+Aliases
+------------------------------------------------------------------
+
+It might be useful to setup a set of shell aliases for the tmt
```

Comment:
As far as I know, `setup` is only used as a noun/adjective but in this instance, you use it as a verb, hence it should be `set up`

Reply:
Thanks for catching this. Fixed.